### ISSUE TITLE
flash fixture - "class" argument name is different in documentation vs implementation

### ISSUE THREAD

**Issue by silvestrenfilho** [web2py/py4web#425]
On documentation, 
    https://py4web.com/_documentation/static/en/chapter-04.html#the-flash-fixture
the example mentions the argument `"_class" `to define a CSS class, whereas the implementation requires argument `"class_"`.

I personally prefer the former, which is consistent with yatl helpers, where named arguments that start with an underscore are interpreted as HTML tag attributes (without the underscore).

**Comment by silvestrenfilho** [comment]
Apparently the required changes to core.py are as follows:
```
py4web$ diff core.py.ori core.py 

350c350
<         flash.set('hello', class_='important')
---
>         flash.set('hello', _class='important')
375c375
<     def set(self, message, class_="", sanitize=True):
---
>     def set(self, message, _class="", sanitize=True):
379c379
<         Flash.local.flash = {"message": message, "class": class_}
---
>         Flash.local.flash = {"message": message, "class": _class}

```

**Comment by nicozanf** [comment]
Hi,

the reference on Documentation is now at https://py4web.com/_documentation/static/en/chapter-06.html?highlight=_class#the-flash-fixture.

Massimo, should we change the Documentation to **class_** or the code to **_class** ?

**Comment by silvestrenfilho** [comment]
It was fixed after I updated to 1.20210906.1

**Comment by silvestrenfilho** [comment]
It was fixed after I updated to 1.20210906.1
Thx
